src/modules/dashboard/components/Dialogs/ManageDomainsDialog/ManageDomainsDialog.tsx

```diff
@@ -35,11 +35,32 @@ const MSG = defineMessages({
     id: 'dashboard.ManageDomainsDialog.domainPermissionsList',
     defaultMessage: 'administration',
   },
+
+  manageWhitelistTitle: {
+    id: 'dashboard.AdvancedDialog.manageWhitelistTitle',
```

Comment:
This should be `.ManageDomainsDialog.` instead of `.AdvancedDialog.`, same as the rest of the ids that were moved